core/templates/services/code-normalizer.service.ts

```diff
@@ -52,12 +52,12 @@ export class CodeNormalizerService {
     var FOUR_SPACES = '    ';
     // Maps the number of spaces at the beginning of a line to an int
     // specifying the desired indentation level.
-    var numSpacesToDesiredIndentLevel = {
+    var numSpacesToDesiredIndentLevel: { [key: number]: number; } = {
```

Comment:
Are you sure about the `;` inside the type definition?
```suggestion
    var numSpacesToDesiredIndentLevel: { [key: number]: number } = {
```